### ISSUE TITLE
Add integration test with a full Docker container

### ISSUE THREAD

**Issue by JosephSalisbury** [giantswarm/inago#207]
None of our current integration tests actually run a Docker container, they're all dummy units. We should add one that runs a Docker container.

**Comment by zeisss** [comment]
Yeah, a more fullblown example might be nice, or just sthg that takes more time.
